backend/pick-git/src/main/java/com/woowacourse/pickgit/post/presentation/PostController.java

```diff
@@ -246,24 +237,18 @@ private PostDeleteRequestDto createPostDeleteRequestDto(AppUser user, Long postI
         @Authenticated AppUser appUser,
         @PathVariable Long postId
     ) {
-        AuthUserForPostRequestDto authUserRequestDto = createAuthUserForPostRequestDto(appUser);
+        AuthUserForPostRequestDto authUserRequestDto =
+            new AuthUserForPostRequestDto(appUser);
```

Comment:
```suggestion
        AuthUserForPostRequestDto authUserRequestDto = new AuthUserForPostRequestDto(appUser);
```

이 부분은 1줄로 작성해도 컨벤션에 맞아요!